src/edfi-paging-test/edfi_paging_test/helpers/argparser.py

```diff
@@ -54,7 +54,7 @@ def parse_main_arguments() -> MainArguments:
         "--connectionLimit",
         help="Maximum concurrent connections to api",
         type=int,
-        default=4,
+        default=5,
```

Comment:
There's definitely some overhead. I tried switching to 10 and it got much worse. 7 was a little better, 8 got worse again. Mileage may vary. 5 seems like a good default.